File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Sys.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Sys {
    @Json(name = "type")
    int type;

    @Json(name = "id")
    int id;

    @Json(name = "country")
    String country;

    @Json(name = "sunrise")
    long sunrise;

    @Json(name = "sunset")
    long sunset;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Sys.kt
package com.riders.thelab.data.remote.dto.weather

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Sys(
    @Json(name = "type")
    val type: Int = 0,

    @Json(name = "id")
    val id: Int = 0,

    @Json(name = "country")
    val country: String,

    @Json(name = "sunrise")
    val sunrise: Long = 0,

    @Json(name = "sunset")
    val sunset: Long = 0,
)
